.appveyor.yml

```diff
@@ -1,36 +0,0 @@
-image: Visual Studio 2019
```

Comment:
There's no reason we _have_ to drop it now. It just doesn't do anything special anymore (just build/test/pack), so figured we could drop it when this merges to `dev`. 

No strong opinion though, happy to revert if you want to keep it for a bit, just let me know.

Reply:
Let's keep Appveyor for now (but without the nupkg publishing to avoid double packages). I just want to see the Github flow working before we fully remove Appveyor, to avoid getting stuck on release etc.